### PR TITLE
Release/v15

### PR DESCRIPTION
**Pull request by benjag**
Release branch for 15.0.0. Individual features should PR against this branch.

### CODE REVIEW

File: src/components/choreographer/CdrChoreographer.vue [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<script setup lang="ts">
```
Comment (KenjiCrosland) [inline src/components/choreographer/CdrChoreographer.vue:1]: I didn't realize we were including this in v15?

File: public/cdr-reset.css [hunk 2]
```diff
@@ -2,12 +2,7 @@ html {
   -webkit-box-sizing: border-box;
   box-sizing: border-box;
   font-family: sans-serif;
-  font-size: 10px; /* update postcss.config.js pxtorem if this changes */
```
Comment (mhewson) [inline public/cdr-reset.css:5]: giphy

Comment (mhewson) [reply public/cdr-reset.css:5]: also some future thoughts https://www.youtube.com/watch?v=cCAtD_BAHNw

File: src/components/abstract/CdrAbstract.vue [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+<template>
+  <component
+    :is="tag"
+    :class="style[baseClass]"
+  >
+    <slot />
+  </component>
+</template>
+
+<style lang="scss" module src="./styles/CdrAbstract.module.scss">
```
Comment (mhewson) [inline src/components/abstract/CdrAbstract.vue:30]: will these also be available in our component variables project?

File: src/components/abstract/styles/CdrAbstract.module.scss [hunk 4]
```diff
@@ -0,0 +1,16 @@
+@import '../../../styles/settings/index.scss';
+@import '../../../styles/settings/fluid.vars.scss';
+
+.cdr-abstract {
```
Comment (mhewson) [inline src/components/abstract/styles/CdrAbstract.module.scss:4]: the tag itself could have a few additional styles I should think

or maybe this is just a global p tag reset 

like 
```
max-width: 75ch;
text-wrap: pretty;
```

File: src/components/choreographer/CdrChoreographer.vue (hunk 1 above)
Comment (benjag) [reply src/components/choreographer/CdrChoreographer.vue:1]: Not exported. Leaving it in for further experimentation

Comment (mhewson) [reply src/components/choreographer/CdrChoreographer.vue:1]: I think the idea is we are not speaking about it

File: src/components/image/CdrImg.vue [hunk 5]
```diff
@@ -1,145 +1,59 @@
...
-  },
-  /**
-   * Image alt text. Defaults to an empty string
-   */
-  alt: {
-    type: String,
-    default: '',
-  },
-  /**
-   * Aspect ratio of the media container
```
Comment (mhewson) [inline src/components/image/CdrImg.vue:28]: 👍

File: src/components/image/CdrImg.vue [hunk 6]
```diff
@@ -1,145 +1,59 @@
...
+      ]); 
+
+/** Checks prop against radiusTokens map for token shorthand and returns if found.
+ *  Otherwise passes value to CSS property */      
+const getRadius = computed(() => radiusTokens.get(props.radius)
+  ? `${radiusTokens.get(props.radius)}rem`
+  : props.radius
+);
+
+const imageProperties = computed(() => {
```
Comment (mhewson) [inline src/components/image/CdrImg.vue:32]: rad, I love the internal custom props

File: src/components/image/styles/CdrImg.module.scss [hunk 7]
```diff
@@ -1,65 +1,10 @@
-@import '../../../styles/settings/_index.scss';
+@import '../../../styles/settings/index.scss';
 
 .cdr-image {
-  max-width: 100%;
-  vertical-align: middle;
-  border-style: none;
-
-  /* Style variants
```
Comment (mhewson) [inline src/components/image/styles/CdrImg.module.scss:8]: amazing

File: src/components/kicker/styles/CdrKicker.module.scss [hunk 8]
```diff
@@ -0,0 +1,9 @@
+@import '../../../styles/settings/index.scss';
+@import '../../../styles/settings/fluid.vars.scss';
+
+.cdr-kicker {
```
Comment (mhewson) [inline src/components/kicker/styles/CdrKicker.module.scss:4]: `text-wrap: nowrap` ?

File: src/components/landingLead/styles/CdrHeadingSubheadingBlock.module.scss [hunk 9]
```diff
@@ -0,0 +1,16 @@
+@import '../../../styles/settings/index.scss';
+@import '../../text/presets/styles/CdrHeadingDisplay.module.scss';
+@import '../../text/presets/styles/CdrSubheadingSans.module.scss';
+
```
Comment (mhewson) [inline src/components/landingLead/styles/CdrHeadingSubheadingBlock.module.scss:4]: lets give our headings and titles 
`text-wrap: balance; `

File: src/components/modal/CdrModal.vue [hunk 10]
```diff
@@ -314,85 +374,87 @@ onUnmounted(() => {
...
-    ref="wrapperEl"
-  >
-    <div :class="[style['cdr-modal__outerWrap'], wrapperClass]">
-      <div
-        aria-hidden="true"
-        @click="onClick"
-        :class="[style['cdr-modal__overlay'], overlayClass]"
-      />
-      <!-- This div (and the one below) is used to avoid a screen reader "keyboard" trap where the
+  <Teleport to="body">
```
Comment (mhewson) [inline src/components/modal/CdrModal.vue:377]: do we need a note for why teleport? I mean I guess its just native Vue...

File: src/components/picture/__tests__/CdrPicture.spec.js [hunk 11]
```diff
@@ -0,0 +1,23 @@
+import { shallowMount } from '../../../../test/vue-jest-style-workaround.js';
+import CdrPicture from '../CdrPicture.vue';
+
+describe('CdrPicture', () => {
+  it('matches snapshot', () => {
+    const wrapper = shallowMount(CdrPicture, {
+      props: {
```
Comment (mhewson) [inline src/components/picture/__tests__/CdrPicture.spec.js:7]: thoughts on how palette could change the image source? Atlassian has and interesting take 
https://atlassian.design/components/image/examples#dark-mode

File: src/components/text/presets/styles/CdrHeadingDisplay.module.scss [hunk 12]
```diff
@@ -0,0 +1,8 @@
+@import './CdrPresets.module.scss';
+
```
Comment (mhewson) [inline src/components/text/presets/styles/CdrHeadingDisplay.module.scss:2]: lets add
`text-wrap: balance; `
to our headings

**Review by mhewson — APPROVED** [review]

File: src/components/abstract/CdrAbstract.vue (hunk 3 above)
Comment (benjag) [reply src/components/abstract/CdrAbstract.vue:30]: No, not at this time. We'd have to figure out providing/exporting the fluid scale to that project

File: src/components/text/styles/CdrText.module.scss [hunk 13]
```diff
@@ -2,6 +2,5 @@
 @import './vars/CdrText.vars.scss';
 
 .cdr-text {
-  @include cdr-text-default;
```
Comment (benjag) [inline src/components/text/styles/CdrText.module.scss:4]: vestigial

**Review by KenjiCrosland — APPROVED** [review]

File: public/cdr-reset.css [hunk 14]
```diff
@@ -38,6 +38,14 @@ body {
   background-color: #ffffff;
 }
 
+h1,h2,h3,h4,h5,h6 {
+  text-wrap: balance;
```
Comment (mhewson) [inline public/cdr-reset.css:42]: 💯

File: public/cdr-reset.css [hunk 15]
```diff
@@ -38,6 +38,14 @@ body {
...
+}
+
+p {
```
Comment (mhewson) [inline public/cdr-reset.css:45]: 🥇

**Pull request merged**